docs/build/guides/transactions/send-and-receive-payments.mdx

```diff
@@ -10,16 +10,16 @@ import Details from "@theme/Details";
 
 ## Payments Overview
 
-A payment constitutes the transfer of a token from one account to another. On Stellar network a token can be either a Stellar asset or a contract token.
+A payment constitutes the transfer of a token from one account to another. On Stellar network a token can be either a Stellar asset or a custom contract token which follows the [SEP-41 token standard](/docs/tokens/stellar-asset-contract#overview).
```

Comment:
@ElliotFriend something something use relative paths for translation to work? I forget the details